src/fr/articles/tableau-accessible.md

```diff
@@ -193,15 +201,15 @@ Le premier exemple est un tableau avec seulement des en-têtes sur les colonnes,
   [...]
 </code></pre>
 
-De cette manière, il est possible de naviguer facilement au sein du tableau à l'aide d'un lecteur d'écran. De plus, tout changement de cellule d'une colonne, ou une ligne, à l'autre, l'en-tête sera vocalisée.
+De cette manière, il est possible de naviguer facilement au sein du tableau à l'aide d'un lecteur d'écran, tout changement de cellule (d'une colonne, ou d'une ligne à l'autre) étant restitué avec l'en-tête associé.
 
-Par Exemple, si on est positionné sur la colonne Prénom, et qu'on utilise le raccourci <kbd>Ctrl+alt+fleche droite</kbd> pour passer à la colonne Nom, NVDA vocalise "Nom Colonne 2 + texte de la colonne".
+Par exemple, si on est positionné sur la colonne "Prénom" et qu'on utilise le raccourci <kbd>Ctrl+Alt+fleche droite</kbd> pour passer à la colonne "Nom", NVDA vocalise _"Nom Colonne 2 + texte de la colonne"_.
 
 ### Tableau à double en-tête
 
-Dans ce deuxième exemple, le tableau est un emploi du temps permettant de savoir rapidement, si le magasin est ouvert ou fermé, en fonction du jour de la semaine et de l'heure.
+Dans ce deuxième exemple, le tableau est un emploi du temps permettant de savoir rapidement si le magasin est ouvert ou fermé, en fonction du jour de la semaine et de l'heure.
 
-Ce tableau nécessite deux en-têtes, une pour les jours de la semaine et une autre pour le créneau horaire.
+Ce tableau nécessite deux en-têtes, un en-tête de colonne pour les jours de la semaine et un en-tête de ligne pour le créneau horaire.
 
 <table class="table">
  <caption class="h4">Ouverture du magasin de jouets</caption>
```

Comment:
The word `jouets` is written without an `s` in the code example just below the `table` (line 259).